opnode/rollup/driver/step.go

```diff
@@ -143,7 +145,7 @@ func (d *outputImpl) step(ctx context.Context, l2Head eth.BlockID, l2Finalized e
 	}
 	// Make batches contiguous
 	minL2Time := l2Info.Time() + d.Config.BlockTime
-	maxL2Time := l1Info.Time()
+	maxL2Time := l1Info.Time() + d.Config.MaxSequencerTimeDiff
```

Comment:
That's required for a constant block time though (one of the bedrock design goals)

Reply:
So part of this is `SortedAndPreparedBatches` isn't a great name for what the function does (I'm thinking of switching it to `FillMissingBatches`), but I'm still pretty sure that this will break things if the `MaxSequencerTimeDiff` isn't set right in comparison to the L1 and L2 blocktimes